docs/userguide/src/SUMMARY.md

```diff
@@ -36,6 +36,8 @@
     - [Performance Tuning](portingguide/perf_tuning/prefix.md)
         - [Link Time Optimization](portingguide/perf_tuning/lto.md)
         - [Optimizing Allocation](portingguide/perf_tuning/alloc.md)
+- [API Migration Guide](migration/prefix.md)
+    - [Template (for mmtk-core developers)](migration/template.md)
```

Comment:
Do we even need to list the template in mdbook? For people who need to edit the migration guide, they will view the template as the raw markdown file rather than the rendered HTML version. For people who do not edit the migration guide, they do not need to see this template in the migration guide at all.

Reply:
I prefer them rendered so that editors know what to expect when looking at the rendered version. But I agree that ordinary readers may not need to read it.  I'll see if we can make it a hidden section.

Reply:
Now I use `mdbook-hide` to hide the chapter, but only in the published version.  It is shown if we build the mdbook locally.